tests/deploy_amm.cairo

```diff
@@ -48,25 +48,63 @@ fn test_deploy_amm() {
     let quote_token_address: ContractAddress = USDC_addr.try_into().unwrap();
     let base_token_address: ContractAddress = ETH_addr.try_into().unwrap();
 
-    deposit(amm_addr, 1, quote_token_address, base_token_address, 0x0583a9d956d65628f806386ab5b12dccd74236a3c6b930ded9cf3c54efc722a1.try_into().unwrap());
-    
-    //trade_option(1705017599, marek_address, amm_addr, FixedTrait::from_unscaled_felt(2200));
+    let amm = IAMMDispatcher { contract_address: amm_addr };
+
+    test_1901_options_added(dispatcher, amm);
+// deposit(amm_addr, 1, quote_token_address, base_token_address, 0x0583a9d956d65628f806386ab5b12dccd74236a3c6b930ded9cf3c54efc722a1.try_into().unwrap());
+
+//trade_option(1705017599, marek_address, amm_addr, FixedTrait::from_unscaled_felt(2200));
+}
+
+fn test_1901_options_added(governance: IGovernanceDispatcher, amm: IAMMDispatcher) {
+    let lp_addresses = amm.get_all_lptoken_addresses();
+    let eth_usdc_call_lp_address = *lp_addresses.at(0);
+    let eth_usdc_put_lp_address = *lp_addresses.at(1);
+    let btc_usdc_call_lp_address = *lp_addresses.at(2);
+    let btc_usdc_put_lp_address = *lp_addresses.at(3);
+
+    // before adding new options
+
+    let eth_usdc_call_options = amm.get_all_options(eth_usdc_call_lp_address);
+    assert(eth_usdc_call_options.len() == 0, 'eth call before');
```

Comment:
This would fail once the amm is deployed, wouldn't it make more sense that it was incremented?